step.yml

```diff
@@ -186,12 +187,13 @@ inputs:
       summary: The maximum number of times a test will repeat based on Test Repetition Mode.
       description: |-
         The maximum number of times a test will repeat based on Test Repetition Mode.
-        
+
         Should be more than 1 if the Test Repetition Mode (`test_repetition_mode`) is other than `none`.
       is_required: true
   - relaunch_tests_for_each_repetition: "no"
     opts:
       title: Relaunch Tests for Each Repetition (Available in Xcode 13+)
+      category: Test Repetition
```

Comment:
Forgot to add `relaunch_tests_for_each_repetition` to the `Test Repetition` category in the previous PR.